asustor_main.c

```diff
@@ -190,6 +192,53 @@ static struct gpiod_lookup_table asustor_as6706_gpio_leds_lookup = {
 	},
 };
 
+static struct gpiod_lookup_table asustor_6300_gpio_leds_lookup = {
+	.dev_id = "leds-gpio",
+	.table = {
+		GPIO_LOOKUP_IDX(GPIO_IT87,   29, NULL,  0, GPIO_ACTIVE_HIGH), // power:front_panel
+		GPIO_LOOKUP_IDX(GPIO_IT87,   59, NULL,  1, GPIO_ACTIVE_HIGH), // power:lcd
+		GPIO_LOOKUP_IDX(GPIO_IT87,   56, NULL,  2, GPIO_ACTIVE_LOW),  // blue:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,    8, NULL,  3, GPIO_ACTIVE_LOW),  // red:power
+		GPIO_LOOKUP_IDX(GPIO_IT87,   31, NULL,  4, GPIO_ACTIVE_LOW),  // green:status
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 20, NULL,  5, GPIO_ACTIVE_HIGH), // red:status
+		// 6
+		GPIO_LOOKUP_IDX(GPIO_IT87,   21, NULL,  7, GPIO_ACTIVE_LOW),  // green:usb
+		GPIO_LOOKUP_IDX(GPIO_IT87,   55, NULL,  8, GPIO_ACTIVE_HIGH), // blue:lan
+
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 23, NULL,  9, GPIO_ACTIVE_HIGH), // sata1:green:disk	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 24, NULL, 10, GPIO_ACTIVE_HIGH), // sata1:red:disk 	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 22, NULL, 11, GPIO_ACTIVE_HIGH), // sata2:green:disk	++
+		GPIO_LOOKUP_IDX(GPIO_AS6300, 21, NULL, 12, GPIO_ACTIVE_HIGH), // sata2:red:disk 	++
+		{}
+		// GPIO_LOOKUP_IDX(GPIO_IT87,   53, NULL, 15, GPIO_ACTIVE_HIGH), // beep:status <<-- It can beep too!!!
```

Comment:
What's the purpose of these commented lines, remove?

Reply:
I guess while probing for LED GPIOs, the author figured out that GPIO 53 beeps and wanted to document this - sensible idea IMHO